docs/src/main/asciidoc/observability-devservices-lgtm.adoc

```diff
@@ -11,7 +11,12 @@ include::_attributes.adoc[]
 :extensions: io.quarkus:quarkus-observability-devservices
 
 This Dev Service provides the https://github.com/grafana/docker-otel-lgtm[Grafana OTel-LGTM], an `all-in-one` Docker image containing an https://opentelemetry.io/docs/collector[OpenTelemetry Collector] receiving and then forwarding telemetry data to https://prometheus.io/[Prometheus] (metrics), https://github.com/grafana/tempo[Tempo] (traces) and https://github.com/grafana/loki[Loki] (logs).
-This data can then be visualized by https://github.com/grafana/grafana[Grafana].
+This data can then be visualized by https://github.com/grafana/grafana[Grafana]. The LGTM abbreviation stands for:
```

Comment:
Probably we should simplify the previous line by removing those links there
Can you please change: 
`receiving and then forwarding telemetry data to https://prometheus.io/[Prometheus] (metrics), https://github.com/grafana/tempo[Tempo] (traces) and https://github.com/grafana/loki[Loki] (logs).`

to 

`receiving and then forwarding telemetry data to Prometheus (metrics), Tempo (traces) and Loki (logs).`